submissions/adeyemiJuwon/database/init.sql

```diff
@@ -0,0 +1,220 @@
+
+
+
+--
+-- Table structure for table `categories`
+--
+
+DROP TABLE IF EXISTS `categories`;
+
+CREATE TABLE `categories` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `category` varchar(100)  NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `categories`
+--
+
+
+INSERT INTO `categories` VALUES (1,'Laptop Accessiories '),
+(2,'Baby Care & Wear'),
+(3,'Women Collection'),
+(4,'Electronic Accessiories'),
+(5,'Smart Home & Garden'),
+(6,'Lights & Lighting');
+
+
+--
+-- Table structure for table `products`
+--
+
+DROP TABLE IF EXISTS `products`;
+
+CREATE TABLE `products` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(50)  NOT NULL,
+  `description` text  NOT NULL,
+  `image` text NOT NULL,
+  `unit_price` decimal(8,2) NOT NULL,
+  `category_id` int(11) NOT NULL,
+  `admin_user_id` int(11) NOT NULL,
+  `quantity_in_stock` int(11) NOT NULL,
+  `stock_level` varchar(50) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `fk_products_admin_users_idx` (`admin_user_id`),
+  KEY `fk_products_categorys_idx` (`category_id`),
+  CONSTRAINT `fk_products_admin_users` FOREIGN KEY (`admin_user_id`) REFERENCES `admin_users` (`id`) ON UPDATE CASCADE,
+  CONSTRAINT `fk_products_categorys` FOREIGN KEY (`category_id`) REFERENCES `categories` (`id`) ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `products`
+--
+
+
+INSERT INTO `products` VALUES (1,'Xiaomi Mi Notebook Pro 2019 15.6 inch Windows 10','Chinese Language Home Edition Intel Core i5 - 8250U Quad Core 1.6GHz CPU 8GB RAM 256GB SSD 1.0MP Front Camera Fingerprint Sensor','load_file(https://samju200.github.io/samju/simple/img/laptop.jpg)',1485.83,1,2,50,'low stock'),
+(2,'Xiaomi Redmibook','Chinese Language Home Edition Intel Core i5 - 8250U Quad Core 1.6GHz CPU 8GB RAM 256GB SSD 1.0MP Front Camera Fingerprint Sensor','load_file(https://samju200.github.io/samju/simple/img/laptop1.jpg)',819.52,1,1,100,'low stock'),
+(3,'Charging power','1.8m DC Charging power ','load_file(https://samju200.github.io/samju/simple/img/charger.jpg)',4.26,1,4,2000,'in stock'),
+(4,'Laptop Bag','11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air','load_file(https://samju200.github.io/samju/simple/img/bag1.jpg)',6.95,1,6,2000,'in stock');
+
+
+--
+-- Table structure for table `customers`
+--
+
+DROP TABLE IF EXISTS `customers`;
+
+CREATE TABLE `customers` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(50) NOT NULL,
+  `last_name` varchar(50)  NOT NULL,
+  `email_address` varchar(250) NOT NULL,
+  `password` varchar(250)  NOT NULL,
+  `phone_number` varchar(50)  NOT NULL,
+  `contact_address` varchar(250)  NOT NULL,
+  `street_address` varchar(250)  NOT NULL,
+  `zip_code` int(11) NULL,
```

Comment:
Just know it is `NULL` by default, you don't have to specify it

Reply:
Okay.
Thank you